examples/tinytest/src/main.rs

```diff
@@ -0,0 +1,137 @@
+extern crate tinyfiledialogs;
+extern crate libc;
+
+use std::ffi::{CString, CStr};
+use libc::c_int;
+use libc::c_uchar;
+
+
+fn message_box(title: &CStr, message: &CStr, dialog_type: &CStr, icon: &CStr, button: c_int) {
+
+    unsafe{
+
+        tinyfiledialogs::tinyfd_messageBox(
+            title.as_ptr(),
+            message.as_ptr(),
+            dialog_type.as_ptr(),
+            icon.as_ptr(),
+            button);
+    }
+}
+fn input_box(title: &CStr, message: &CStr, default: &CStr) {
+
+    unsafe{
+
+        tinyfiledialogs::tinyfd_inputBox(title.as_ptr(),message.as_ptr(),default.as_ptr());
```

Comment:
nit: add a space after each , when there are multiple arguments on the same line.